Dockerfile

```diff
@@ -14,7 +14,9 @@ RUN addgroup --gid 2000 --system appgroup && \
 WORKDIR /app
 
 COPY --chown=appuser:appgroup build/libs/dps-data-compliance*.jar /app/app.jar
+COPY --chown=appuser:appgroup build/libs/applicationinsights-agent*.jar /app/agent.jar
 COPY --chown=appuser:appgroup run.sh /app
+COPY --chown=appuser:appgroup AI-Agent.xml /app
```

Comment:
you've added the agent to the dockerfile, but can't see a change to `run.sh` where you're calling it.
Better still, just ditch `run.sh` and change
```
ENTRYPOINT ["java", "-javaagent:/app/agent.jar", "-jar", "/app/app.jar"]
```
then it's clearer what is going on